pkg/telemetry/phonehome/runtime_config.go

```diff
@@ -17,42 +17,49 @@ const (
 	selfManagedKey = "eDd6QP8uWm0jCkAowEvijOPgeqtlulwR"
 )
 
-type remoteConfig struct {
-	Key string `json:"storage_key_v1,omitempty"`
+// RuntimeConfig defines some runtime features.
+type RuntimeConfig struct {
```

Comment:
For reviewers' attention: no `version` field. It can be added later, when there is an incompatible change.